03_more_datatypes/2_lists/03_09_flatten.py

```diff
@@ -13,3 +13,9 @@
 '''
 
 starting_list = [[1, 2, 3, 4], [5, 6], [7, 8, 9]]
+list_lenght=len(starting_list)
+result=[]
+for i in range(int(list_lenght)):
+    a=starting_list[i]
+    result=result+a
```

Comment:
well done, this could be written a bit more pythonically.


```suggestion
result=[]
for sublist in starting_list:
    result += sublist
```

This gets you the same answer, but is a bit cleaner and easier to use.
As time goes you will get a better idea when you need to do a for-loop with range (and use the index) vs just do a for loop directly on the object.